src/test/java/org/opensearch/security/auditlog/compliance/ComplianceAuditlogTest.java

```diff
@@ -56,6 +56,7 @@ public void testSourceFilter() throws Exception {
         Settings additionalSettings = Settings.builder()
             .put("plugins.security.audit.type", TestAuditlogImpl.class.getName())
             .put(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_TRANSPORT, true)
+            .put(ConfigConstants.SECURITY_SYSTEM_INDICES_ADDITIONAL_CONTROL_ENABLED_KEY, false)
```

Comment:
Looks like many tests were updated, like this one, why this change?

Reply:
The system already defaults to false, but i thought would be better to be explicit on the tests that are expecting it to be false.

Reply:
These tests that were update don't seem to have any new behavior or assertions, why add this change?

Reply:
Ah, I see what you mean now. Those were in tests that i was debugging and made changes before the flag was added but didnt not notice that i had reverted my changes on them.  I removed the referenced lines